Server/models/page.js

```diff
@@ -0,0 +1,29 @@
+const { DataTypes } = require('sequelize');
+const sequelize = require('../util/database');
+
+const Page = sequelize.define('Page', {
+    PageNumber: {
```

Comment:
Switch the properties between PageNumber and PageId. PageId should be the primary key, autoincrement, etc. PageNumber is just a number.